docs/en/observability/synthetics-quickstart.asciidoc

```diff
@@ -21,118 +23,96 @@ and a packaged todo application with a custom suite of tests.
 
 [discrete]
 [[synthetics-quickstart-step-one]]
-== Step 1: Clone the `elastic/synthetics` repository
-
-This template pulls the Elastic synthetics image, builds your synthetic tests,
-schedules and runs them with Heartbeat, and sends the resulting data to the Elastic Stack.
+== Step 1: Pull the latest synthetics docker image
 
-Clone the https://github.com/elastic/synthetics[elastic/synthetics] repository
-and change directories into `examples/docker`:
+Elastic Synthetics is regularly updated during this experimental phase, even within the same version.
+You'll want to ensure you're running the latest docker image by running the below command.{synthetics-image}
 
-[source,sh]
+[source,sh,subs="attributes"]
 ----
-git clone https://github.com/elastic/synthetics.git &&\
-cd synthetics/examples/docker
+docker pull {synthetics-image}
 ----
 
-There are two files in `synthetics/examples/docker` that you'll need to edit:
-
-[source,sh]
-----
-synthetics
-  |- examples
-     |- docker
-        |- heartbeat.docker.yml <1>
-        |- run.sh <2>
-----
-<1> `heartbeat.docker.yml` is your Heartbeat configuration file.
-This is where you'll configure your synthetic test suites.
-<2> `run.sh` provides the main `docker run` command that pulls the
-Elastic synthetics image and runs Heartbeat.
 
 [discrete]
 [[synthetics-quickstart-step-two]]
-== Step 2: Update `heartbeat.docker.yml`
+== Step 2: Create a heartbeat.yml configuration file
 
 There are two ways to configure a synthetic test.
 Let's take a quick look at each.
 
 Run an inline test::
 
-If you're running an inline, browser based test, you can use the traditional Heartbeat flow to completely
-configure your synthetic testing directly in `heartbeat.docker.yml`.
-An example is provided:
-+
+If you're running an inline browser based test, you can use the traditional Heartbeat flow to completely
+configure your synthetic testing directly in a file named `heartbeat.yml` as shown below. 
+
+You'll need to setup your elasticsearch credentials for either an https://www.elastic.co/guide/en/beats/heartbeat/current/configure-cloud-id.html[Elastic Cloud ID] or another https://www.elastic.co/guide/en/beats/heartbeat/current/elasticsearch-output.html[Elasticsearch Cluster] in
+the `heartbeat.yml` file. The example below demonstrates using https://cloud.elastic.co[Elastic Cloud] credentials.
+
 [source,yml]
 ----
 heartbeat.monitors:
 - type: browser
   id: my-monitor <1>
   name: My Monitor
   schedule: "@every 1m"
-  script: |- <2>
-    step("load homepage", async () => {
-        await page.goto('https://www.elastic.co');
-    });
-    step("hover over products menu", async () => {
-        await page.hover('css=[data-nav-item=products]');
-    });
+  source:
+    inline:
+      script: |- <2>
```

Comment:
Is the `|-` still needed?

Reply:
Yes, that syntax lets you use multiline strings in yaml